server/app/controllers/admin/AdminProgramBlocksController.java

```diff
@@ -34,8 +35,11 @@
 /** Controller for admins editing screens (blocks) of a program. */
 public final class AdminProgramBlocksController extends CiviFormController {
 
+  private static final boolean READ_ONLY = true;
```

Comment:
**suggestion (blocking):** With its current usage this variable doesn't look necessary